testutils/clustermanager/gke.go

```diff
@@ -228,19 +212,17 @@ func (gc *GKECluster) Acquire() error {
 		err = nil
 
 		clusterName := request.ClusterName
-		// Deleting cluster if it already exists
+		// Use cluster it already exists and running
 		existingCluster, _ := gc.operations.GetCluster(*gc.Project, region, request.Zone, clusterName)
-		if existingCluster != nil {
-			log.Printf("Cluster %q already exists in region %q zone %q. Deleting...", clusterName, region, request.Zone)
-			err = gc.operations.DeleteCluster(*gc.Project, region, request.Zone, clusterName)
+		if existingCluster != nil && existingCluster.Status == ClusterRunning {
+			gc.Cluster = existingCluster
+			return nil
 		}
-		// Creating cluster only if previous step succeeded
-		if err == nil {
-			log.Printf("Creating cluster %q in region %q zone %q with:\n%+v", clusterName, region, request.Zone, gc.Request)
-			err = gc.operations.CreateCluster(*gc.Project, region, request.Zone, rb)
-			if err == nil { // Get cluster at last
-				cluster, err = gc.operations.GetCluster(*gc.Project, region, request.Zone, rb.Cluster.Name)
-			}
+		// Creating cluster
+		log.Printf("Creating cluster %q in region %q zone %q with:\n%+v", clusterName, region, request.Zone, gc.Request)
+		err = gc.operations.CreateCluster(*gc.Project, region, request.Zone, rb)
+		if err == nil { // Get cluster at last
```

Comment:
Can we rephrase the comment or may be just delete it?

Reply:
removed